src/lib/contents/docs/compatibility-matrix.ts

```diff
@@ -2,199 +2,467 @@ import type { Matrix } from "$lib/types/matrix.type";
     components: [
       {
         name: "Kubernetes",
         availibility: "supported",
+        limitations: null,
         policy: {
-          text: "the last minor three releases",
+          text: "the last three minor releases",
           description: null,
         },
         supportedVersions: "1.22, 1.23 and 1.24",
       },
+    ],
+  },
+  {
+    name: "Orchestration Platform", // RESPONSIBLE TEAM: WORKSPACE
+    relevance: "self-hosted admins", // this is optional used for filtering based on relevance.
+    components: [
       {
-        name: "Elastic Kubernetes Service (EKS)",
-        availibility: "not-supported",
+        name: "Amazon Elastic Kubernetes Service (EKS)",
+        availibility: "supported",
+        limitations: null,
+        policy: {
+          text: "see Kubernetes support policy",
+          description: null,
+        },
+        supportedVersions: "see Kubernetes support policy",
+      },
+      {
+        name: "Google Kubernetes Engine (GKE)",
+        availibility: "supported",
+        limitations:
+          '- <a href="https://cloud.google.com/kubernetes-engine/docs/concepts/dataplane-v2">Dataplane V2</a> unsupported, <br/> - Not supported in combination with <a href="https://github.com/gitpod-io/gitpod/issues/11168">self-signed certs</a>',
         policy: {
-          text: "default",
+          text: "see Kubernetes support policy",
           description: null,
         },
-        supportedVersions: "default",
+        supportedVersions: "see Kubernetes support policy",
       },
       {
-        name: "Elastic Kubernetes Service (EKS)",
+        name: "Azure Kubernetes Service (AKS)",
         availibility: "supported",
+        limitations: null,
+        policy: {
+          text: "see Kubernetes support policy",
+          description: null,
+        },
+        supportedVersions: "see Kubernetes support policy",
+      },
+      {
+        name: "K3s",
+        availibility: "untested",
+        limitations: null,
         policy: {
-          text: "default",
-          description: "",
+          text: "see Kubernetes support policy",
+          description: null,
         },
-        supportedVersions: "default",
+        supportedVersions: "see Kubernetes support policy",
       },
     ],
   },
   {
-    name: "Container Network Interface (CNI)",
-    relevance: "admins",
+    name: "Container Runtime", // RESPONSIBLE TEAM: WORKSPACE
+    relevance: "self-hosted admins",
     components: [
       {
-        name: "Kubernetes",
+        name: "containerd",
         availibility: "supported",
+        limitations: "- Socket access required",
         policy: {
-          text: "the last minor three releases",
+          text: "Last three minor releases",
           description: null,
         },
-        supportedVersions: "1.22, 1.23 and 1.24",
+        supportedVersions: "1.4, 1.5 and 1.6.",
```

Comment:
I don't think we support containerd 1.4 anymore.
cc @aledbf

Reply:
> I don't think we support containerd 1.4 anymore.

We don't, and it's EOL since [March 3, 2022](https://github.com/containerd/containerd/blob/main/RELEASES.md#support-horizon)

Also, we are not using 1.5 in core-dev or SaaS either.

Reply:
Again, which versions do we support then? :)

Reply:
```suggestion
        supportedVersions: "1.5 and 1.6.",
```

I say this only because we use the [unspecified channel](https://github.com/gitpod-io/gitpod/blob/b09a3eb796230b3accbe590358d53b4ddb57e166/install/infra/modules/gke/main.tf#L87) to provision self-hosted infrastructure, [default to using 1.22](https://github.com/gitpod-io/gitpod/blob/2cd2b3e51fadfe151ab334041b4acbe95f2b19ab/install/infra/single-cluster/gcp/variables.tf#L38), and [both 1.22 channels](https://cloud.google.com/kubernetes-engine/docs/release-notes#current_versions) support containerd 1.5.

@lucasvaltl How should we communicate that 1.5 should work, but, it is not covered as well as 1.6?
1. integration tests (for workspace) are failing against preview environments, even with 1.6, [we are working to fix and it should also fix for 1.5](https://github.com/gitpod-io/gitpod/issues/12248).
2. integration test results for self-hosted infrastructure are not being shared with product teams with 1.5. Can you share how these are being shared now?
3. as @aledbf indicated, core-dev and saas use 1.6, and we do not support 1.4

Reply:
Happy to merge this and try to state that 1.5 is not tested. This also clashes with the versioning policy here ("Last three minor release"). Should we change the policy?

Reply:
@kylos101  ping on this - should I change the versioning policy to the last two releases? Or fix it at 1.5 and 1.6?

Reply:
Implemented a change here that changes both the policy and the currently supported versions to just `1.5 and 1.6`. Also added a note making it clear that we prefer 1.6 which is what we use internally. Same as below:  note that when we do make changes to the version we support here, we will have to update both. This also does not give customers much predictability - but I am fine merging this like this.